### PR TITLE
fix(rate):use constant

### PR DESCRIPTION
**Pull request by Notryag**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/rate/src/index.vue [hunk 1]
```diff
@@ -48,7 +48,7 @@ import { isObject, isArray } from '@vue/shared'
 import { hasClass } from '@element-plus/utils/dom'
 import { EVENT_CODE } from '@element-plus/utils/aria'
 import { elFormKey } from '@element-plus/form'
-
+import { UPDATE_MODEL_EVENT } from '@element-plus/utils/constants'
```
Comment (JeremyWuuuuu) [inline packages/rate/src/index.vue:51]: Add a empty line between import members and import types

**Review by jw-foss — APPROVED** [review]
I am happy to approve as-is

**Comment by JeremyWuuuuu** [comment]
@Notryag Sorry to ping you here again.. I've made a huge change into this project, you might need to resolve the conflicts :bow:

**Comment by Notryag** [comment]
> @Notryag Sorry to ping you here again.. I've made a huge change into this project, you might need to resolve the conflicts 🙇

Is this ok

**Review by jw-foss — APPROVED** [review]

**Pull request merged**